src/components/NavBar/index.js

```diff
@@ -126,7 +127,7 @@ const DivSearchBarField = styled.div`
   width: 60%;
 `;
 
-const SearchBarField = ({ intl, classes, store: { search }, onClose, onSearchBarKeyDown }) => (
+const SearchBarField = ({ intl, classes, store: { search, ui }, onSearchBarKeyDown }) => (
```

Comment:
should inject the store like you did with the `SearchButton`. be consistent with your coding practices.

Reply:
Do we also need to do the same way with Withstyles/Intl inject on these components?
And I saw some code at navbar components also using the similar way to access the store, should I also modify them as well?

Reply:
@ippudo ya we should be consistent across all stateless components. if we inject store or withStyles in one component, we should do it with the others. don't worry about refactoring all the ones that aren't touched in this PR. can do that later.